src/api/spec/support/files/request_payload_github_pull_request_closed.txt

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This extension of this file and also for similar files in this PR should be renamed  from `.txt` to `.json`.

Reply:
Done!